src/Source/AbstractPropertySourceIterator.php

```diff
@@ -155,7 +155,7 @@ protected function getCurrentData($current): array
     /**
      * @param mixed $value
      *
-     * @return string|null
+     * @return bool|int|float|string|null
```

Comment:
In the `default` case, we are returning the same value received in the argument 1. Should we use the same types in its `@param` docblock?

Reply:
They are included in mixed. And I think we're supporting other types.
For instance, object are casted to string.